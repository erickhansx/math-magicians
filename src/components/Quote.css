.quote {
  display: flex;
  text-align: center;
  margin: 0 auto;
  justify-content: center;
  align-items: center;
  min-height: 60vh;
}

.quote__text {
  font-size: 26px;
}
