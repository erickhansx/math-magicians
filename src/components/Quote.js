function Quote() {
  return (
    <div className="quote">
      <p className="quote__text">
        Mathematics is not about numbers, equations , computations, or
        algorithms: it is about understanding. -William Paul Thurston
      </p>
    </div>
  );
}

export default Quote;
