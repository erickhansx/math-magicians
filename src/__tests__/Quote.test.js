import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

test('Quote Renders', () => {
  const quoteRender = renderer.create(<Quote />).toJSON();
  expect(quoteRender).toMatchSnapshot();
});
