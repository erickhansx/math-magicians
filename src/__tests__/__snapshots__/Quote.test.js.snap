// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote Renders 1`] = `
<div
  className="quote"
>
  <p
    className="quote__text"
  >
    Mathematics is not about numbers, equations , computations, or algorithms: it is about understanding. -William Paul Thurston
  </p>
</div>
`;
